Source/Runtime/Tenancy/Tenants.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+syntax = "proto3";
+
+import "Fundamentals/Protobuf/Uuid.proto";
+import "Fundamentals/Protobuf/Failure.proto";
+
+package dolittle.runtime.tenancy;
+
+option csharp_namespace = "Dolittle.Runtime.Tenancy.Contracts";
+option go_package = "go.dolittle.io/contracts/runtime/tenancy";
+
+message GetAllRequest {
+    // TODO: Do we want another kind of execution context here?
```

Comment:
Since this is not a management request - I think it would make sense to use the `CallRequestContext` here, even though we're not going to use the tenant-id for anything. In practice, it would probably be set to `TenantId.System`, but the other information in the ExecutionContext might prove useful at some point. Would be nice if all the SDK->Runtime calls had the same request context.

Reply:
That is true, we should have that